packages/core/lib/utils.js

```diff
@@ -217,19 +222,22 @@ var utils = {
       var pair = header.split('=');
 
       if (pair[0] && pair[1]) {
-        var key = pair[0].trim().toLowerCase();
-        var value = pair[1].trim().toLowerCase();
-        var reserved = reservedKeywords.indexOf(key) !== -1;
+        var key = pair[0].trim();
+        var value = pair[1].trim();
+        var lowerCaseKey = key.toLowerCase();
+        var reserved = reservedKeywords.indexOf(lowerCaseKey) !== -1;
```

Comment:
Is it necessary to create a new `lowerCaseKey` variable instead of converting the key & value to lowercase using `.toLowerCase`?

nit: use `let` instead of `var`.

Reply:
I don't want to mutate the casing of additional trace data since it may contain a case-sensitive key. So I created a separate variable `lowerCaseKey` from `key` as the former is used in multiple places below for existing logic.
Also, I don't think we need to change the casing of the value anywhere.